deploy/backplane/srep/30-cee-pcap-collector.Role.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: backplane-srep-pcap-collector
+  namespace: openshift-backplane-managed-scripts
+rules:
+# can create cred secret for mustgathers
+# Enables https://github.com/openshift/managed-scripts/blob/main/scripts/CEE/pcap-collector/metadata.yaml, which requires users to be able to create/get/delete the pcap-collector-creds secret in the managed scripts ns
+- verbs:
+    - "create"
```

Comment:
Doesn't this secret need to get deleted afterwards?

Reply:
@cblecker the secret is deleted by script (no matter whether it fails or whether it succeeds: https://github.com/openshift/managed-scripts/pull/97/files#diff-30ff287cfece93b43e84f8f9a0f3da6e8e520a064d068a4f1866953e4702f886R167). It's probably good to allow users to delete the secret, too, if they've made a mistake in creating the secret and want to re-create it. I've amended the pull request to reflect this.

Reply:
@wanghaoran1988 recommended that we only keep the 'create' permission - but I don't mind either way. I see a use case for the 'delete' permission being there, too. https://github.com/openshift/managed-cluster-config/pull/1376#discussion_r1042969504

Reply:
I didn't see that before. Yeah I think permitting deletion in the case of mistake/cleanup is probably worthwhile. What do you think @wanghaoran1988 ?

Reply:
@cblecker Make sense to me